src/layout/Input/InputComponent.tsx

```diff
@@ -24,21 +25,33 @@ export function InputComponent({
     formData?.simpleBinding ?? '',
     saveWhileTyping,
   );
-
   const handleChange = (e) => setValue(e.target.value);
 
   return (
-    <TextField
-      id={id}
-      onBlur={saveValue}
-      onChange={handleChange}
-      onPaste={onPaste}
-      readOnly={readOnly}
-      isValid={isValid}
-      required={required}
-      value={value}
-      aria-describedby={textResourceBindings?.description ? `description-${id}` : undefined}
-      formatting={formatting as IInputFormatting}
-    />
+    <>
+      {variant === 'search' ? (
+        <SearchField
+          id={id}
+          value={value}
+          onChange={handleChange}
+          onBlur={saveValue}
+          onPaste={onPaste}
+          aria-describedby={textResourceBindings?.description ? `description-${id}` : undefined}
```

Comment:
I think you lost the `readOnly` property on the way here. It is named `disabled` in `SearchField`.

Reply:
Måtte synce med det nyeste fra main og da kjørte Cypress testene lokalt :)